plugins/juju/juju.plugin.zsh

```diff
@@ -0,0 +1,131 @@
+# ---------------------------------------------------------- #
+# Aliases and functions for juju (https://juju.is)           #
+# ---------------------------------------------------------- #
+
+# Load TAB completions
+# You need juju's bash completion script installed. By default bash-completion's                    
+# location will be used (i.e. pkg-config --variable=completionsdir bash-completion).
+() {
+  local completion_file="$(pkg-config --variable=completionsdir bash-completion)/juju" ||
+    local completion_file="/usr/share/bash-completion/completions/juju"
+  [[ -f "$completion_file" ]] && source "$completion_file"
+}
```

Comment:
The `||` is never executed because `local` commands always return true, you could define a normal variable and then that works:
```suggestion
completion_file="$(pkg-config --variable=completionsdir bash-completion 2>/dev/null)/juju" || \
  completion_file="/usr/share/bash-completion/completions/juju"
[[ -f "$completion_file" ]] && source "$completion_file"
unset completion_file
```